src/tests/match-experiences.test.ts

```diff
@@ -21,9 +20,12 @@ const testMatchExperience: MatchExperiencePayload = {
     stadium: 'Wembley Stadium',
     title: 'Exciting Match!',
     description: 'This will be an intense match between top teams.',
+    likes: [],
+    createdAt: new Date(),
+    updatedAt: new Date(),
```

Comment:
```suggestion
    likes: [],
    createdAt: new Date(),
    updatedAt: new Date(),
```
I think you can remove all of those and it works just as fine